app/views/layouts/application.html.slim

```diff
@@ -9,5 +9,5 @@ html
         = yield
         = any_login_here if defined?(AnyLogin)
       #footer
-    = render 'layouts/footer'
+      = render 'layouts/footer'
```

Comment:
No, this layout uses sticky footer from Compass library, so we need use this markup: http://compass-style.org/reference/compass/layout/sticky_footer/